setup.py

```diff
@@ -119,6 +119,13 @@ def run(self):
         sdist.run(self)
 
 
+efilter_version = __version__
+if "bdist_msi" in sys.argv and efilter_version[1] == "!":
+  # bdist_msi does not support a setuptools epoch version so we remove it
+  # as a work around.
+  _, _, efilter_version = efilter_version.partition("!")
```

Comment:
Alas the setuptools versioning scheme is not supported by its build targets. The same already applies to the filename of the tar.gz sdist creates. What's you concern?